packages/bot-web-ui/src/components/toolbox/toolbox.jsx

```diff
@@ -0,0 +1,192 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Field as FormField, Formik, Form } from 'formik';
+import { Drawer, Input, Icon, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { ToolboxItems } from './toolbox-items.jsx';
+import { connect } from '../../stores/connect';
+import { popover_zindex } from '../../constants/z-indexes';
```

Comment:
Do we have an alias for `constants`, if so, can we use that here?

Reply:
We don't have absolute imports in bot-web-ui package atm. But yeah, it'll be good if we can add it later.